Cargo.toml

```
@@ -20,8 +20,8 @@ repository = "https://github.com/0xPolygonMiden/miden-client"
 [workspace.dependencies]
 async-trait = "0.1"
 miden-lib = { version = "0.7", default-features = false }
-miden-objects = { version = "0.7", default-features = false }
-miden-tx = { version = "0.7", default-features = false, features = ["async"] }
+miden-objects = {  version = "0.7.2", default-features = false }
```

We can drop the patch version from this and just `cargo update` miden-base dependencies

---

also nit (for `miden-tx` as well), the extra space can be dropped as well:
```suggestion
miden-objects = { version = "0.7", default-features = false }
```